Mark samples with valid recovered genomes but no genome submission info as gisaid-rejected

ttung opened the pull request:
### Description
Add a failed workflow to signify a failed submission.

Depends on #377 

### Test plan
will import and test

shanaxel42 reviewed (approved):
Ahhhh the rejected case

Commit:
Mark samples with valid recovered genomes but no genome submission info as gisaid-rejected

### Description
Add a failed workflow to signify a failed submission.

### Test plan
will import and test
